qiskit-superstaq/qiskit_superstaq/superstaq_backend.py

```diff
@@ -138,5 +161,9 @@ def run(
         return job
 
     def target_info(self) -> Dict[str, Any]:
-        """Returns backend's target information."""
+        """Returns Superstaq backend's target information.
```

Comment:
"backend's target information" is a little confusing imo. wdyt about changing it to just
```suggestion
        """Returns information about this backend.
```
? (might be a more clear/consistent way to phrase this though..)

Reply:
Sounds good!